README.md

```diff
@@ -36,6 +36,7 @@ because we use this endpoint for importing existing data.
 > - `psql -U postgres clarin-utilities < <UTILITIES_DUMP_FILE_PATH>`
 
 or use `scripts/start.local.dspace.db.bat`.
+NOTE: `start.local.dspace.db.bat` and `init.dspacedb5.sh` must have `LF` line separator.
```

Comment:
nope, bat must have CRLF
sh must have LF
ensure this using
https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings

```
*.sh eol=lf
*.py eol=lf
*.md eol=lf
*.bat eol=crlf
```